docs/carrier.md

```diff
@@ -215,6 +215,34 @@ To ensure compatibility, set the switch manufacturer to `bandwidth.com`.
 [carriers-main]: /carrier/img/carriers-main.png "Carriers"
 [carrier]: /carrier/img/carrier.png "Carrier Main"
 
+## Headers from Customers to Providers
+
+When the Carrier requires information on the headers sent from the UAC, you can use below discussed process:
```

Comment:
Customize the handling of X-Headers between the User Agent Client (UAC) and the carrier. By default, all UAC-originating X-Headers are removed for standardization, but this configuration allows selective retention of specific headers crucial for carrier communication.

Reply:
added in the introduction